NOTICE.BINARY

```diff
@@ -926,99 +486,15 @@ Other developers who have contributed code are:
 
 
 
-================= joda-time-2.9.9.jar =================
-=============================================================================
-= NOTICE file corresponding to section 4d of the Apache License Version 2.0 =
-=============================================================================
-This product includes software developed by
-Joda.org (http://www.joda.org/).
-
-
-
-
-================= log4j-1.2-api-2.5.jar =================
-Apache Log4j 1.x Compatibility API
-Copyright 1999-2015 Apache Software Foundation
-
-
-
-
-================= log4j-api-2.5.jar =================
-Apache Log4j API
-Copyright 1999-2015 Apache Software Foundation
-
-
-
-
 ================= log4j-core-2.5.jar =================
 Apache Log4j Core
 Copyright 1999-2012 Apache Software Foundation
 
 ResolverUtil.java
 Copyright 2005-2006 Tim Fennell
 
-
-
-
-================= log4j-jul-2.5.jar =================
-Apache Log4j JUL Adapter
-Copyright 1999-2015 Apache Software Foundation
-
-
-
-
-================= log4j-slf4j-impl-2.5.jar =================
-Apache Log4j SLF4J Binding
-Copyright 1999-2015 Apache Software Foundation
-
-
-
-
-================= maven-aether-provider-3.1.1.jar =================
-Maven Aether Provider
-Copyright 2001-2013 The Apache Software Foundation
-
-
-
-
-================= maven-artifact-3.6.0.jar =================
-Maven Artifact
-Copyright 2001-2018 The Apache Software Foundation
-
-
-
-
-================= maven-model-3.1.1.jar =================
-Maven Model
-Copyright 2001-2013 The Apache Software Foundation
-
-
-
-
-================= maven-model-builder-3.1.1.jar =================
-Maven Model Builder
-Copyright 2001-2013 The Apache Software Foundation
-
-
-
-
-================= maven-repository-metadata-3.1.1.jar =================
-Maven Repository Metadata Model
-Copyright 2001-2013 The Apache Software Foundation
-
-
-
-
-================= maven-settings-3.1.1.jar =================
-Maven Settings
-Copyright 2001-2013 The Apache Software Foundation
-
-
-
-
-================= maven-settings-builder-3.1.1.jar =================
-Maven Settings Builder
-Copyright 2001-2013 The Apache Software Foundation
+TypeUtil.java
```

Comment:
for `log4j-core-2.5.jar`, I don't see ResolverUtil or TypeUtil class accessible from our project dependencies, so I think this can be removed as well